docs/workflow/building/coreclr/android.md

```diff
@@ -1,15 +1,64 @@
-Cross Compilation for Android on Linux
-======================================
+# Experimental support of CoreCLR on Android
+
+This is the internal documentation which outlines experimental support of CoreCLR on Android and includes instructions on how to:
+- [Build CoreCLR for Android](./android.md#building-coreclr-for-android)
+- [Build and run a sample Android application with CoreCLR](./android.md#building-and-running-helloandroid-sample-app)
+- [Debug the sample app and the runtime](./android.md#debugging-the-sample-app)
+
+## Prerequisite
+
+Download and install `Android Studio` and the following:
+  - Android SDK (minimum supported API level is 21)
+  - Android NDK r27
+
+## Building CoreCLR for Android
+
+Supported host systems for building CoreCLR for Android:
+- [MacOS](./android.md#macos) ✔
+- [Linux](./android.md#linux) ✔
+- [Windows](./android.md#windows) ❌ (only through WSL)
+
+Supported target architectures:
+- x86 ❌
+- x64 ✔
+- arm ❌
+- arm64 ✔
+
+### MacOS
+
+#### Requirements
+
+Set the following environment variables:
+  - ANDROID_SDK_ROOT=`<full-path-to-android-sdk>`
+  - ANDROID_NDK_ROOT=`<full-path-to-android-ndk>`
+
+For example:
+  ```
+  ANDROID_SDK_ROOT=/Users/<user>/Library/Android/sdk
+  ANDROID_NDK_ROOT=/Users/<user>/Library/Android/sdk/ndk/27.2.12479018
+  ```
+
+#### Building the runtime, libraries and tools
+
+To build CoreCLR runtime packages, libraries and tools, run the following command from `<repo-root>`:
+
+```
+./build.sh clr.runtime+clr.alljits+clr.corelib+clr.nativecorelib+clr.tools+clr.packages+libs+packs -os android -arch <x64|arm64> -c <Debug|Release> -bl
```

Comment:
The build of `+packs` subset seems to be broken at the moment. The native hosts are excluded for build in some places but not in others. That will likely need to be fixed but in the meantime we should match the documentation with the actual test scripts (which do not build the `packs` subset).

Reply:
Updated the instructions for building the NuGet packages